### PR TITLE
Additional FIPS Checksums CI fixes

### PR DESCRIPTION
**Pull request by t8m**
The providers/common/der/*.in were not included correctly.

The fips-checksums CI workflow was not working properly when checking pristine and merged sources into the same directory.

### CODE REVIEW

File: Configurations/unix-Makefile.tmpl [hunk 1]
```diff
@@ -1227,7 +1221,8 @@ providers/fips.module.sources.new: \
 		   crypto/ec/asm/*.pl \
 		   crypto/modes/asm/*.pl \
 		   crypto/sha/asm/*.pl \
-		   crypto/x86_64cpuid.pl; do \
+		   crypto/x86_64cpuid.pl \
+	           providers/common/der/*.in; do \
```
Comment (paulidale) [inline Configurations/unix-Makefile.tmpl:1225]: Much neater than my solution.

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
CI doesn't look relevant.

**Comment by t8m** [comment]
> CI doesn't look relevant.

Yep. I also cannot reproduce it.

**Comment by t8m** [comment]
Merged to master. Thank you for the review.

**Pull request closed without merging**